### DISCUSSION TITLE
🚀 Distro 2021.11.1 Released

### DISCUSSION THREAD

**Discussion by Nicogene** [robotology/community#570] [Releases]
Hi @robotology/everyone 👋,

We're happy to announce that the latest iCub **Software Distro 2021.11.1** is now available. 📦 

To download the source code with the correct tags, check out our up-to-date [Software Versioning Table](https://icub-tech-iit.github.io/documentation/sw_versioning_table/2021.11.1/).

This distro is coming with the release of iCub S/N 048 −  iCubValparaiso01. 🤖

Like all the patch releases, it is shipped only in terms of tags/branches, no binaries are provided.

This release contains the configuration files of `iCubValparaiso01` and some important fixes/changes listed below: 

- Fixed the initialization of imu bosch bno055 in the RFE board firmware (see robotology/icub-firmware#230).
- Ported the iCub 3 configuration files to the use of the latest NWS/NWC layers (see robotology/robots-configuration#319).
- YARP-3.6 released https://github.com/robotology/yarp/releases/tag/v3.6.0.
- Added explicit dependency of `bipedal-locomotion-framework` to `OsqpEigen` (see robotology/robotology-superbuild#952)

**Comment by pattacini** [comment]
Another important feature is the possibility to expose within the `yarpmotorgui` the latest joint fault state.

See:
- https://github.com/robotology/yarp/pull/2764.
- https://github.com/robotology/icub-main/pull/779.
